src/api.ts

```diff
@@ -1,13 +1,19 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=5a03dcb7';
 
-export function getMovie(query: string): Promise<MovieData | ResponseError> {
-  return fetch(`${API_URL}&t=${query}`)
-    .then(res => res.json())
-    .catch(() => ({
+export async function getMovie(
+  query: string,
+): Promise<MovieData | ResponseError> {
+  const response = await fetch(`${API_URL}&t=${query}`);
+
+  try {
+    return await response.json();
+  } catch {
+    return ({
       Response: 'False',
       Error: 'unexpected error',
-    }));
+    });
```

Comment:
use throw new Error instead of returning such object
in this way you'll be available to handle this error in catch {} while calling this function